contrib/native/client/src/clientlib/channel.cpp

```diff
@@ -211,6 +211,21 @@ ChannelContext* ChannelFactory::getChannelContext(channelType_t t, DrillUserProp
             }
 
             pChannelContext = new SSLChannelContext(props, tlsVersion, verifyMode);
+
+            if (props->isPropSet(USERPROP_CUSTOM_SSLCTXOPTIONS)){
```

Comment:
This is an optional settings that the client application can set since the default configuration from boost is somewhat limited. And also based on the version of boost the client is compiled against, some configurations are not available.